watertap/unit_models/gac.py

```diff
@@ -172,6 +173,21 @@ class GACData(UnitModelBlockData):
         **SurfaceDiffusionCoefficientType.calculated** - calculates surface diffusion coefficient}""",
         ),
     )
+    CONFIG.declare(
+        "target_species",
+        ConfigValue(
+            default=None,
+            domain=set,
```

Comment:
Not sure what difference it would make here, but why use `domain=set` instead of `domain=dict`?

Reply:
Here, I was just mirror how the sets are configured in the property package such as solute set, and wanted to be able to use set operators in later code (see inert_species). In future model development I could see the whole solute_set or ion_set getting passed into the target_species which I think would make this a more seamless transition to higher fidelity models.